pom.xml

```diff
@@ -1587,40 +1589,69 @@
       </build>
     </profile>
 
+    <!-- hudi-spark2-bundle is an alias for hudi-spark-bundle -->
     <profile>
       <id>spark2</id>
+      <properties>
+        <sparkbundle.version>2</sparkbundle.version>
+      </properties>
+      <activation>
+        <property>
+          <name>spark2</name>
+        </property>
+      </activation>
+    </profile>
+
+    <profile>
+      <id>spark2.x</id>
```

Comment:
What's the diff b/w spark2 and spark2.x?

Reply:
Originally spark2 is activated by default, generating hudi-spark-bundle (no number).  I now keep it as is and only have one new profile of spark2.4 generating hudi-spark2.4-bundle.  There is no point of generating hudi-spark2-bundle as it's not there before.